finality-aleph/src/block/substrate/mod.rs

```diff
@@ -91,7 +94,8 @@ impl BlockImport<Block> for BlockImporter {
             import_existing: false,
             state: None,
         };
-        self.metrics.report_block(hash, Checkpoint::Importing);
+        self.metrics
+            .report_block(hash, Checkpoint::Importing, Some(number), Some(own));
         self.importer.import_blocks(origin, vec![incoming_block]);
```

Comment:
This is a different behavior than we've had before - we change `origin`of imported block that is passed to the actual importer. Are we sure this is correct?

Reply:
I'm afraid `FinalityRateMetrics` require this change. Somehow they have to get the knowledge that the block was produced by us. Unless you have some other suggestions..

Reply:
Ok, then lets leave it. Last change related to that is to update docs of the function import_block in src/sync/handler/mod.rs at line 476.

Reply:
ok, thx for catching the doc update - fixed it.